src/VisionModel/convnb.jl

```diff
@@ -0,0 +1,46 @@
+# Simple rewrite of Conv without biases, usefull for convolutional layer followed by BatchNorm
```

Comment:
Since this package is fairly new, I am fine excluding this file and depending on changes in Flux#master. It will be some time before this package is registered, and I expect another Flux release before then.

Reply:
I can remove this workaround, however I need it because Flux#master causes errors on my system (with Image.jl in particular).
For me it is important to perform quick tests on CIFAR10, to see if model is learning properly.

Reply:
All right. I’ll just remove the workaround after we merge.

Reply:
I just succeed to install Flux#master (it conflict with metalhead), so I will remove the workaround today.

Reply:
Ah I just merged this PR. Want to submit another once you remove the workaround 😉 ?

Reply:
Sure. Can you tell me how I merge my fork with the original. Should I do a new fork?

Reply:
One option is to delete the fork and make a new one, but the better way is [to rebase from upstream](https://www.atlassian.com/git/tutorials/git-forks-and-upstreams). If you follow that guide to add this repo's master branch as the upstream, then rebase from it (not merge), you should be good. You will have the latest copy of this repo's master and it would be like creating a new fork. Then you can submit a new PR when you are ready like you did before.

Reply:
Thank you